### PR TITLE
pod: add websockets e2e benchmark

### PR DESCRIPTION
**Pull request by kckeiks**
Adds websockets benchmark.

## Checklist

- [x] I have made corresponding changes to the tests
- [ ] ~I have made corresponding changes to the documentation~
- [ ] ~I have run the app using my feature and ensured that no functionality is broken~

### CODE REVIEW

File: crates/ursa-pod/benches/e2e.rs [hunk 1]
```diff
@@ -302,14 +315,180 @@ mod tcp_tls_ufdp {
...
+                                io::ErrorKind::Other,
+                                e.to_string(),
+                            )))
+                        }
+                        None => return Poll::Ready(Ok(())),
+                    });
+            };
+
+            let mut buff = vec![0; item_to_copy.remaining().min(buf.remaining())];
+            item_to_copy.read(&mut buff).unwrap();
```
Comment (ozwaldorf) [inline crates/ursa-pod/benches/e2e.rs:477]:
```suggestion
            item_to_copy.read_exact(&mut buff).unwrap();
```

**Review by ozwaldorf — COMMENTED** [review]
overall looks good some minor suggestions

**Review by qti3e — APPROVED** [review]
lgtm

File: crates/ursa-pod/benches/e2e.rs (hunk 1 above)
Comment (ozwaldorf) [reply crates/ursa-pod/benches/e2e.rs:473]: Dont forget this guy!

**Pull request merged**